### PR TITLE
Feature/windows compatibility

### PR DESCRIPTION
**Pull request by shnewto**
Exposes bustype parameter for Windows users and bitrate parameter for additional configuration.

Prior to this commit using this project on a Windows machine
required editing the source code. After this commit the required
bustype configuration is exposed on the command line. This commit
also exposes the bitrate parameter and adds documentation to specify
the required usagages for both a Linux and Windows system.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,19 +91,60 @@ python3 setup.py install --user
...
+Options:
+    -h --help                            Display this information
+    -d --disable                         Disable modules only, no further checks (overrides enable)
+    -e --enable                          Enable modules only, no further checks checks
+    -l --loop                            Repeat all checks, run continuously
+    -b --bustype <bustype>               CAN bus type [default: socketcan_native]
+                                         (for more see https://python-can.readthedocs.io/en/2.1.0/interfaces.html)
+    -B --bitrate <bitrate>               CAN bus bit rate [default: 500000]
+    -c <channel>, --channel <channel>    Specify CAN channel, [default: can0]
+    -V <vehicle>, --vehicle <vehcile>    Specify your vehcile, [default: kia_soul_ev]
```
Comment (adrienthebo) [inline README.md]: typo - `vehcile` -> `vehicle`

File: README.md [hunk 2]
```diff
@@ -87,19 +91,60 @@ python3 setup.py install --user
...
-                                     (kia_soul_ev / kia_soul_petrol / kia_niro)
--v --version                         Display version information
+Options:
+    -h --help                            Display this information
+    -d --disable                         Disable modules only, no further checks (overrides enable)
+    -e --enable                          Enable modules only, no further checks checks
+    -l --loop                            Repeat all checks, run continuously
+    -b --bustype <bustype>               CAN bus type [default: socketcan_native]
+                                         (for more see https://python-can.readthedocs.io/en/2.1.0/interfaces.html)
+    -B --bitrate <bitrate>               CAN bus bit rate [default: 500000]
```
Comment (adrienthebo) [inline README.md]: I haven't seen an instance where the bitrate isn't `500000` - is there a concrete scenario where this is different? If not, can we omit this feature until the need presents itself? My thinking is that if we can omit this functionality, that's a bit less code we need to maintain. :)

File: README.md [hunk 3]
```diff
@@ -87,19 +91,59 @@ python3 setup.py install --user
 
 ## Usage
 
-`oscc-check.py [-hdelv] [-c <channel>] [-V <vehicle>]`
+`oscc-check.py [-hdelv] [-b <bustype>] [-B <bitrate>] [-c <channel>] [-V <vehicle>]`
```
Comment (cotos) [inline README.md]: I think you want to take the -B option out of usage.

File: README.md [hunk 4]
```diff
@@ -87,19 +91,59 @@ python3 setup.py install --user
...
--v --version                         Display version information
+Options:
+    -h --help                            Display this information
+    -d --disable                         Disable modules only, no further checks (overrides enable)
+    -e --enable                          Enable modules only, no further checks checks
+    -l --loop                            Repeat all checks, run continuously
+    -b --bustype <bustype>               CAN bus type [default: socketcan_native]
+                                         (for more see https://python-can.readthedocs.io/en/2.1.0/interfaces.html)
+    -c <channel>, --channel <channel>    Specify CAN channel, [default: can0]
+    -V <vehicle>, --vehicle <vehcile>    Specify your vehcile, [default: kia_soul_ev]
```
Comment (adrienthebo) [inline README.md]: Another `vehcile` misspelling :)

File: README.md [hunk 5]
```diff
@@ -72,10 +72,14 @@ But what does it do?
 
 Works with OSCC `v1.2.1` and up.
 
-### Dependencies
+### Linux Dependencies
 
 - `python3` (`sudo apt install python3`)
-- `pip3` (`sudo apt install python3-pip`)
```
Comment (rebpdx) [inline README.md]: Are there any dependencies on imported things like the color terminal output that need a pip install?

**Pull request merged**